TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -222,7 +222,7 @@ private TableOrViewAllColumnsSelection CreateSelection(CodeElementsParser.Select
         }
 
         private SymbolReference CreateSymbolReference(Token nameToken, Token qualifierToken,
-            Token topLevelQualifierToken)
+            Token topLevelQualifierToken = null)
```

Comment:
Why the topLevelQualifier deserves a default value ? I mean, the qualifier is optional too